Lib/pdb.py

```diff
@@ -1889,34 +1996,34 @@ def set_trace(*, header=None):
 
 # Post-Mortem interface
 
-def post_mortem(t=None):
-    """Enter post-mortem debugging of the given *traceback* object.
+def post_mortem(tb_or_exc=None, /):
+    """Enter post-mortem debugging of the given *traceback*, or *exception*
+    object.
 
     If no traceback is given, it uses the one of the exception that is
     currently being handled (an exception must be being handled if the
     default is to be used).
+
+    If `t` is an exception object, the `exceptions` command makes it possible to
+    list and inspect its chained exceptions (if any).
     """
     # handling the default
-    if t is None:
+    if tb_or_exc is None:
         exc = sys.exception()
         if exc is not None:
-            t = exc.__traceback__
+            tb_or_exc = exc.__traceback__
```

Comment:
I _guess_ we do not need this anymore, and below we need to update the error message ?